src/Typesense.php

```diff
@@ -123,9 +123,14 @@ public function deleteDocument(Collection $collectionIndex, $modelId): array
         /**
          * @var $document Document
          */
-        $document = $collectionIndex->getDocuments()[(string) $modelId];
+        $document = $collectionIndex->getDocuments()[(string) $modelId] ?? null;
```

Comment:
Due to how `getDocuments` works, this has been problematic in the past, so we'd need to remove the `?? null` (https://github.com/typesense/laravel-scout-typesense-driver/issues/7)